website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -56,7 +61,10 @@ var NewAndNoteworthy = {
                 self.popularNodes().push(result.data[l]);
                 self.fetchContributors(result.data[l]);
             }
-            self.someDataLoaded(true);
+            // NOTE: This manual redraw is necessary because we set background: true on
+            // the request, which prevents a redraw. This redraw allows the loading
+            // indicator to go away and the first 10 nodes to be rendered
+            m.redraw();
```

Comment:
I would remove.  Discover Public Projects are only two requests.  One to fetch new and noteworthy and one to fetch most popular.   Removing these prevents the javascript error.